crates/ty_ide/src/goto_definition.rs

```diff
@@ -41,6 +41,95 @@ mod test {
     use ruff_db::files::FileRange;
     use ruff_text_size::Ranged;
 
+    /// goto-definition on a module should go to the .py not the .pyi
+    /// FIXME: this currently doesn't work right!
```

Comment:
nit: our usual practice in ty is TODO comments, not FIXME comments (assuming you aren't planning to fix this before landing this PR)